### PR TITLE
Dockerfile.fedora: install packages to build catatonit

### PR DESCRIPTION
**Pull request by vrothberg**
Needed to update the CI images for #4090.

Signed-off-by: Valentin Rothberg <rothberg@redhat.com>

### CODE REVIEW

**Comment by mheon** [comment]
LGTM

File: Dockerfile.fedora [hunk 1]
```diff
@@ -15,6 +18,7 @@ RUN dnf -y install btrfs-progs-devel \
               libassuan-devel \
               libseccomp-devel \
               libselinux-devel \
+              libtool \
```
[the remark this answers is not in the document]
Comment (vrothberg) [reply Dockerfile.fedora]: Very good point. Yes, we need that there too. I can add that to  #4090.

**Pull request merged**